src/SourceBuild/content/repo-projects/format.proj

```diff
@@ -1,10 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <ItemGroup>
-    <RepositoryReference Include="arcade" />
```

Comment:
Should keep arcade here.

Reply:
Fixed in https://github.com/dotnet/installer/pull/18557/commits/608e3adab65aca27adcc828a17383c6b401466dc